citus.spec

```diff
@@ -71,12 +72,14 @@ echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
 %files
 %defattr(-,root,root,-)
 %doc CHANGELOG.md
+
 %if 0%{?rhel} && 0%{?rhel} <= 6
 %doc LICENSE
 %else
 %license LICENSE
 %endif
 %doc %{pginstdir}/doc/extension/README-%{sname}.md
+%doc /usr/NOTICE
```

Comment:
I do not like this generic file name that can cause clashes with many other packages.

Reply:
Fixed as you suggested